promo_wheel/index.html

```diff
@@ -0,0 +1,407 @@
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M12.3366 28.0778C12.7691 27.8078 22.9241 21.3541 22.9241 12.0178C22.9204 9.03526 21.7339 6.17588 19.6249 4.06688C17.5159 1.9579 14.6567 0.771461 11.6741 0.767822C8.69151 0.771461 5.83213 1.9579 3.72313 4.06688C1.61415 6.17588 0.42771 9.03526 0.424072 12.0178C0.424072 21.3541 10.5791 27.8078 11.0116 28.0778L11.6741 28.4928L12.3366 28.0778ZM11.6741 27.3128L11.807 27.2295C11.986 27.1178 14.5496 25.4917 17.0571 22.7701C19.5773 20.0346 21.9236 16.3213 21.9241 12.019C21.9208 9.30129 20.8396 6.69576 18.9178 4.77399C16.9964 2.85252 14.3915 1.77146 11.6741 1.76782C8.95675 1.77146 6.35172 2.85253 4.43024 4.77399M4.43024 4.77399C2.50863 6.69561 1.42754 9.3009 1.42407 12.0185C1.4243 16.3209 3.77074 20.0345 6.29108 22.7701C8.79858 25.4917 11.3621 27.1178 11.5411 27.2295L11.5424 27.2304L11.6741 27.3128M11.6741 18.2678C10.4379 18.2678 9.22957 17.9013 8.20176 17.2144C7.17395 16.5277 6.37287 15.5516 5.89982 14.4096C5.42677 13.2676 5.30301 12.0109 5.54416 10.7985C5.78532 9.58612 6.38057 8.47249 7.25466 7.59841C8.12873 6.72432 9.24232 6.12907 10.4547 5.88791C11.6672 5.64676 12.9238 5.77052 14.0658 6.24357C15.2078 6.71662 16.1839 7.5177 16.8707 8.54551C17.5576 9.57331 17.9241 10.7817 17.9241 12.0178C17.9221 13.6748 17.2629 15.2633 16.0913 16.4351C14.9196 17.6067 13.3311 18.2658 11.6741 18.2678ZM6.54755 6.89131C5.53361 7.90524 4.84312 9.19706 4.56337 10.6034C4.28364 12.0098 4.4272 13.4675 4.97595 14.7923C5.52468 16.117 6.45393 17.2493 7.6462 18.0459C8.8385 18.8427 10.2402 19.2678 11.6741 19.2678H11.6753C13.5971 19.2655 15.4394 18.501 16.7984 17.1422C18.1572 15.7832 18.9218 13.9408 18.9241 12.019V12.0178C18.9241 10.584 18.499 9.18224 17.7022 7.98995M6.54755 6.89131C7.56147 5.87738 8.85324 5.18688 10.2596 4.90713C11.6661 4.6274 13.1238 4.77095 14.4485 5.3197C15.7732 5.86842 16.9055 6.79771 17.7022 7.98995"
+              fill="white"
+            />
+          </svg>
+        </div>
+        <div class="wheel">
+          <div class="one">15% OFF</div>
+          <div class="two">10% OFF</div>
+          <div class="three">30% OFF</div>
+          <div class="four">20% OFF</div>
+          <div class="five">25% OFF</div>
+          <div class="six">Try Again</div>
+        </div>
+        <div class="wheel promo-code-code-message">
+          <h2 class="deal"></h2>
+          <p>your next order</p>
+          <p>Use Code</p>
+          <h1 class="promo-code"></h1>
+        </div>
+      </div>
+      <button class="shop-now-button">Shop Now</button>
+      <form>
+        <input
+          type="email"
+          name="email-address"
+          placeholder="Enter your Email"
+          required
+        />
+        <input
+          type="submit"
+          value="Try My Luck!"
+          data-onesignal-unique-label="submit-button"
+        />
+      </form>
+    </div>
+    <script type="text/javascript">
+      window.deal = "";
+      window.promoCode = "";
```

Comment:
if we just assign these with `var` won't they be in the global scope vs assigning them directly to the window?

Reply:
that's true